pybamm/parameters/lead_acid_parameters.py

```diff
@@ -771,6 +762,16 @@ def a_p(self, x):
         x_dim = x * self.L_x
         return self.a_p_dimensional(x_dim) / self.a_p_typ
 
+    def rho(self, T):
```

Comment:
Due to the definition of the current collectors this is not equal to one (as it is with lead-acid...).